### DISCUSSION TITLE
Trying to initiate a oauth flow callback to retrieve etsy outstanding orders and get this CAPTCHA triggered server response

### DISCUSSION THREAD

**Discussion by Clarry100** [etsy/open-api#1306] [Q&A]
Etsy Server response from oauth flow call: 

403 - <html><head><title>etsy.com</title><style>#cmsg{animation: A 1.5s;}@keyframes A{0%{opacity:0;}99%{opacity:0;}100%{opacity:1;}}</style></head><body style="margin:0"><p id="cmsg">Please enable JS and disable any ad blocker</p><script data-cfasync="false">var dd={'rt':'c','cid':'AHrlqAAAAAMAsGGNWP6_ItQAMbDcOw==','hsh':'D013AA612AB2224D03B2318D0F5B19','t':'bv','s':45249,'e':'6a962b532ba7a0e6f618385ebe40222de2f92afaed8680e3e7974dee6fdbd8a9','host':'geo.captcha-delivery.com','cookie':'0aevCXgBUNvy1QjVHsGTZPfAedSE4Q3RnMfCFrD~dcwCIDK87uWgHhEVfejuKZdmlUHYUgLFa6aL1xukMYmmrPmS1uVPbz_fBTqedh53HAf7tKOk96v3v0RS7ObA_sVl'}</script><script data-cfasync="false" src="https://ct.captcha-delivery.com/c.js"></script></body></html>

This is my very first attempt at developing an Etsy API call using oauth, so be kind to me. LOL.

I'm definitely not a guru of all things IT but this looks like etsy are flagging my oauth call request as a security risk and hence requesting a CAPTCHA response. As I understand these kinda responses cannot be done programmatically and must be done via human response.

Just wondering if anyone else developing etsy APIs using oauth have encountered such a server response and how you ended up solving this.

**Comment by Wilderbill** [comment]
Ask ChatGPT for its opinion.  :-)

**Reply by Clarry100** [reply]
I did... it says ask for help on either these pages or the etsy help centre. Etsy just keep on deleting the post unanswered after a couple of weeks. And nobody on here seems to have an answer. So I dunno where to turn to now.

**Comment by Wilderbill** [comment]
I was not able to get past the OAuth process until I found this:
    https://github.com/slider1234/ETSY-API-Auto-Authenticator

**Reply by Clarry100** [reply]
Oh yeah. Thanks. Will look into this one. Cheers
